firebase-vertexai/README.md

```diff
@@ -30,3 +30,8 @@ the [spotless plugin]([https://plugins.jetbrains.com/plugin/14912-ktfmt](https:/
 by running:
 
 `./gradlew firebase-vertexai:spotlessApply`
+
```

Comment:
Actually, it should be

https://github.com/googleapis/googleapis/blob/master/google/cloud/aiplatform/v1/prediction_service.proto

Reply:
TBH I think it would be safer to not link to the protos